index.html

```diff
@@ -139,8 +139,8 @@ <h2 class="tonic3-b">Facebook 360</h2>
       </div>
     </section>
     <section class="tonic2">
-      <img src="images/multipost.png" alt="tonic" class="mob">
-      <img src="imagesdesktop/toni2-4.png" alt="tonic" class="desk desk2">
+      <img src="images/multipost.png" alt="tonic4" class="mob">
+      <img src="imagesdesktop/toni2-4.png" alt="tonic4" class="desk desk2">
```

Comment:
- [ ] [Optional] Even though they play the same roles but different screens, these images should have different `alt` values